pdns/rec_channel.cc

```diff
@@ -54,6 +54,12 @@ int RecursorControlChannel::listen(const string& fname)
   if(bind(d_fd, (sockaddr*)&d_local,sizeof(d_local))<0) 
     throw PDNSException("Unable to bind to controlsocket '"+fname+"': "+stringerror());
 
+  // receive buf should be size of max datagram plus address size
+  int bufsz = 60*1024;
```

Comment:
Out of curiosity, why 60k for the send buffer and 64k for the receive one?

Reply:
At least on OpenBSD, the receive buffer must be bigger than the send buffer, since it is also used to receive the peer address internally. This might not be true on all systems, though.